### PR TITLE
Local correlation dimension

### PR DESCRIPTION
**Pull request by Datseris**

### CODE REVIEW

**Comment by Datseris** [comment]
@PythagoreanCult Can you add a test here? Or, just test this function with some example set of points and confirm that it does what we expect?

**Review by PythagoreanCult — APPROVED** [review]

**Comment by Datseris** [comment]
@PythagoreanCult please don't approve PRs without actually testing them. I just wrote a test for the code and it doesn't run. It errors:

```
      
ERROR: UndefVarError: `Eucliean` not defined
Stacktrace:
 [1] local_correlation_sum(X::StateSpaceSet{2, Float64, SVector{2, Float64}}, ζ::SVector{2, Float64}, k::Int64)
   @ FractalDimensions C:\Users\datse\.julia\dev\FractalDimensions\src\corrsum_based\correlationsum_vanilla.jl:245
 [2] local_correlation_dimension(::StateSpaceSet{…}, ::Vararg{…}; fit::LinearRegression, kw::@Kwargs{})
   @ FractalDimensions C:\Users\datse\.julia\dev\FractalDimensions\src\corrsum_based\correlationsum_vanilla.jl:240
 [3] local_correlation_dimension(::StateSpaceSet{2, Float64, SVector{2, Float64}}, ::Vararg{Any})
   @ FractalDimensions C:\Users\datse\.julia\dev\FractalDimensions\src\corrsum_based\correlationsum_vanilla.jl:239
```

there are several typos. Next time please write and run an actual test before approving or not approve at all if you are out of time. For now I've written and pushed a test here so we're all good.

**Pull request merged**
